### PR TITLE
ENT-5767 : Kotlin 1.3/1.4 abstract classes/methods are not instrumented correctly in quasar

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by josephzunigadaly — APPROVED** [review]

File: quasar-kotlin/src/test/kotlin/co/paralleluniverse/kotlin/fibers/lang/AbstractTest.kt [hunk 1]
```diff
@@ -0,0 +1,149 @@
+/*
+ * Quasar: lightweight threads and actors for the JVM.
+ * Copyright (c) 2015-2017, Parallel Universe Software Co. All rights reserved.
+ *
```
Comment (blsemo) [inline quasar-kotlin/src/test/kotlin/co/paralleluniverse/kotlin/fibers/lang/AbstractTest.kt:4]: Is this a new file? If yes, it should be Copyright 2020 R3 Ltd...

**Review by blsemo — COMMENTED** [review]
Looks good as far as I understand quasar, but question about the copyright headers.

File: quasar-kotlin/src/test/kotlin/co/paralleluniverse/kotlin/fibers/lang/AbstractTest.kt (hunk 1 above)
Comment (PaulHatcherR3) [reply quasar-kotlin/src/test/kotlin/co/paralleluniverse/kotlin/fibers/lang/AbstractTest.kt]: Yes, I've just looked through Corda code base. I could not find a suitable pattern to use. One for the Guilds maybe.
I've removed this Copyright notice for now.

**Review by chrisr3 — APPROVED** [review]
OK

**Pull request merged**